### PR TITLE
Send command styling (#437)

### LINKED ISSUE #437: Transaction messages missing styling
Transaction messages should be styled  as per (bottom of page)
https://zpl.io/Z1cQdF4

### PR DESCRIPTION
**Pull request by rasom**
fix #437

### CODE REVIEW

File: ios/StatusIm.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -45,6 +44,8 @@
 		82E689BAF9FB43C8AC6FF1CA /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = CEB0E2659D1A4F5FA842057A /* EvilIcons.ttf */; };
 		832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 832341B51AAA6A8300B99B32 /* libRCTText.a */; };
 		8E55E6877F950B81C8D711C5 /* libPods-StatusIm.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 101A4045637A2ADF57D28EF5 /* libPods-StatusIm.a */; };
+		9E0B01811DDC5CFD002B0359 /* libSplashScreen.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 9E0B017E1DDC5CD1002B0359 /* libSplashScreen.a */; };
```
Comment (alwx) [inline ios/StatusIm.xcodeproj/project.pbxproj]: why do you have a splash screen changes in this commit?

**Comment by alwx** [comment]
yep, it is works fine in develop now

**Comment by rasom** [comment]
Changes related to SplashScreen are removed.

**Comment by alwx** [comment]
Looks fine. Merge it.

**Pull request merged**